recipes/libint2/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "libint" %}
+{% set version = "2.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/evaleev/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tgz
+  sha256: 253fa3301e6742f5aac9e96dbe2a5707fc73abd6f4a214a8d44f03e113e69f00
+
+build:
+  number: 0
+  binary_relocation: true
+  skip: true  # [not linux]
```

Comment:
I think macOS should work again now if you want to try and re-enable it.

If you do, make sure to only use `-fopenmp` on Linux as it's not supported by clang.

Reply:
thanks!

Reply:
[macOS keeps on failing](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=13695&view=logs)

Reply:
I can't see the the error as the build log has been cleaned. (I guess because the a new commit appeared before all parts of the Azure pipeline completed?)

It can be Linux only for now but in case you want to add macOS support in future: Travis failed due to a timeout which you can avoid in the actual feedstock [by using CircleCI instead](https://conda-forge.org/docs/ciservices.html?highlight=provider#using-circle-for-both-linux-and-osx).

Reply:
It is OK if is Linux for Now? The typical application for this library is a scientific simulation that run in a HPC facilities that use Linux as OS

Reply:
Yeah that's fine so far as I'm concerned 👍